packages/widgets/package.json

```diff
@@ -127,17 +127,13 @@
     "validate-commit-msg": "^2.12.2"
   },
   "optionalDependencies": {
-    "create-inferno-context": "^0.2.4",
-    "create-react-context": "^0.2.3",
-    "inferno": "^7.1.9",
-    "inferno-compat": "^7.1.9",
     "preact": "^8.4.2",
     "preact-compat": "^3.18.4",
     "preact-context": "^1.1.3",
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
-    "@builder.io/react": ">=1.1.0"
+    "@builder.io/react": ">=1.1.0 ^2.0.2-5"
```

Comment:
isn't this syntax broken? It doesn't make sense to me

if you go to this calculator: https://semver.npmjs.com/

and put the values in, you see only ^2.0.2-5 as highlighted 
<img width="987" alt="image" src="https://user-images.githubusercontent.com/1393142/179238101-de352f35-ac16-4700-ab8d-250074499d6c.png">

```suggestion
    "@builder.io/react": "^2.0.2-5"
```

Reply:
It's valid, but it's only temporary, the original one `>=1.1.0` should be unchanged, this was needed because `>=` doesn't include dev versions, I'll remove it once a patch version of the react sdk is released